microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -1,70 +1,88 @@
 ---
-description: This article provides documentation on the Microsoft Edge user agent client hints and user agent string
-title: How to detect Microsoft Edge in your website
+description: This article describes how to detect Microsoft Edge data with user agent client hints and user agent string
+title: Detecting Microsoft Edge from your website
 author: MSEdgeTeam
-ms.author: msedgedevrel
-ms.date: 05/19/2021
+ms.author: v-vchapel
+ms.date: 06/22/2021
 ms.topic: article
 ms.prod: microsoft-edge
-keywords: microsoft edge, compatibility, web platform, user agent string, ua string, ua overrides, user-agent client hints, user agent client hints, ua client hints, ua ch
+keywords: microsoft edge, compatibility, web platform, user agent string, ua string, ua overrides, user-agent client hints, user agent client hints, ua client hints, ua ch, feature detection, browser identification, browser detection, header, https header, verify chromium, detect microsoft edge, detecting microsoft edge
 ---
-# How to detect Microsoft Edge in your website  
+# Detecting Microsoft Edge from your website

```

Comment:
Might want to change the toc.yml entry too

Reply:
Oh of course, changed, thanks for letting me know!